clinica/iotools/bids_utils.py

```diff
@@ -114,6 +115,8 @@ def create_participants_df(
     for i in range(0, len(participant_df)):
         if study_name == "OASIS":
             value = (participant_df["alternative_id_1"][i].split("_"))[1]
+        elif study_name == "OASIS3":
+            value = re.sub("OAS3", "", participant_df["alternative_id_1"][i])
```

Comment:
You should be able to use the `replace`method from string instead of a regexp substitution.